### PR TITLE
Users/jack/container workspace dir

### PR DESCRIPTION
**Pull request by jackjia-ibm**
Sort out workspace mount point.

### CODE REVIEW

File: containers/conformance.md [hunk 1]
```diff
@@ -173,29 +173,30 @@ In the runtime, the Zowe content is organized in this structure:
...
         +- /<component-id>
     +- /instance
-      +- instance.env or zowe.yaml
+      +- zowe.yaml
       +- /logs
-      +- /tmp
       +- /workspace
     +- /keystore
-      +- zowe-certificates.env
+      +- keystore.p12
```
Comment (1000TurquoisePogs) [inline containers/conformance.md:180]: I don't think these files should be guaranteed to be here.
What we could say is "keystore content will be in the /keystore directory"
whether thats a p12, a cer, or both is really up to the users and would be specified in the zowe.yaml

File: containers/kubernetes/workloads/api-catalog-deployment.yaml [hunk 2]
```diff
@@ -100,20 +100,20 @@ spec:
...
-              mountPath: "/home/zowe/logs"
+              mountPath: "/home/zowe/instance/logs"
             - name: zowe-workspace
-              mountPath: "/home/zowe/workspace"
+              mountPath: "/home/zowe/instance/workspace"
       initContainers:
         - name: init-zowe
           # image: zowe-docker-release.jfrog.io/ompzowe/zowe-launch-scripts:2-ubuntu
-          image: zowe-docker-snapshot.jfrog.io/ompzowe/zowe-launch-scripts:2.0.0-ubuntu.v2-x-staging
+          image: zowe-docker-snapshot.jfrog.io/ompzowe/zowe-launch-scripts:2.0.0-ubuntu.users-jack-container-workspace-dir
```
Comment (1000TurquoisePogs) [inline containers/kubernetes/workloads/api-catalog-deployment.yaml:116]: keeping this or going back to v2.x/staging?

File: containers/conformance.md (hunk 1 above)
Comment (jackjia-ibm) [reply containers/conformance.md]: Agreed. changes wording.

File: containers/kubernetes/workloads/api-catalog-deployment.yaml (hunk 2 above)
Comment (jackjia-ibm) [reply containers/kubernetes/workloads/api-catalog-deployment.yaml]: It's for testing purpose. Will be reverted before merge.

**Pull request merged**